services/src/handlers/wms.rs

```diff
@@ -149,9 +154,8 @@ fn get_capabilities(_request: &GetCapabilities) -> Result<HttpResponse> {
             <Format>BLANK</Format>
         </Exception>
         <Layer queryable="1">
-            <Name>Test</Name>
-            <Title>Test</Title>
-            <CRS>EPSG:4326</CRS>
```

Comment:
I chose not to include the projection because you can query the layers in any projection and it is automatically reprojected.

for WFS I think I forgot it. Shall I remove it too?